test/testScript.js

```diff
@@ -0,0 +1,136 @@
+/** This script is used to test all commands in testengine. It will run most operations in testengine-cli with a few flag
+ * combinations and print it to screen. A tester can then read the output to see how the cli behaves. Together with the
+ * script there are a few readyapi-projects and login configurations for testengine. Assumes the testengine instance is
+ * running at localhost:8080 and has the logins admin/password, regularUser/asdasd.
+ * To run the script you need node.js. Ensure the working directory is the test-folder and run:
+ * node testScript.js
+ */
+
+const {execSync, exec} = require('child_process');
+
+const baseCli = 'node ../bin/testengine.js';
+const licenseServer = 'localhost:1194';
+
+function startJob(projectPath) {
+    let startJob = [baseCli, 'run project', projectPath, '-c admin.config'].join(' ');
+    let extractTestJobId = baseCli + " jobs list -c admin.config -c admin.config|sed -n 3p |sed 's/ *$//g'|rev|cut -d ' ' -f 2|rev"
+    exec(startJob)
+    let jobId = execSync(extractTestJobId).toString().trim()
+    return jobId
+}
+
+function runAllCombinations(commands, flags, fn) {
+    for (let i = 0; i < commands.length; ++i) {
+        for (let j = 0; j < flags.length; ++j) {
+            fn(commands[i], flags[j])
+        }
+    }
+}
+
+function runCli(command, flag, jobId = '') {
+    let cli = [baseCli, command, jobId, flag].join(' ');
+    console.log('\n' + cli);
+    try {
+        console.log(execSync(cli).toString() + 'exit status 0');
+    } catch (error) {
+        console.log('' + error.stdout + 'exit status ' + error.status);
+    }
+}
+
+
+function startSlowJobAndThenRunCli(command, flag) {
+    let testJobId = startJob('slow.xml')
+    runCli(command, flag, testJobId)
```

Comment:
semicolon missing. There are other places as well where semicolon is missing. Please take a look

Reply:
fixed